src/Controls/src/Core/Layout/Layout.cs

```diff
@@ -137,8 +137,7 @@ public void Clear()
 			{
 				if (this[index] is Element element)
 				{
-					element.Parent = null;
-					VisualDiagnostics.OnChildRemoved(this, element, index);
+					OnChildRemoved(element, index);
```

Comment:
What happens to the existing code? Is it called somewhere else or was this a double clear and visual diagnostics call?

Reply:
It's not getting reparented right? so no need to remove from the LVT?

Reply:
These two lines (the clearing of the parent and the removal from the diagnostics tree) are both called in `Element.OnChildRemoved()`.

(Along with some other bookkeeping stuff that we were missing by not just calling `OnChildRemoved()`).